Code/Editor/Plugins/EditorAssetImporter/ImporterRootDisplay.cpp

```diff
@@ -65,7 +69,15 @@ void ImporterRootDisplay::SetSceneHeaderText(const QString& headerText)
 {
     QFileInfo fileInfo(headerText);
     ui->m_filePathText->setText(fileInfo.fileName());
-    ui->m_fullPathText->setText(QString("%1%2").arg(QDir::toNativeSeparators(fileInfo.path())).arg(QDir::separator()));
+    QString fullPath = QString("%1%2").arg(QDir::toNativeSeparators(fileInfo.path())).arg(QDir::separator());
+    ui->m_fullPathText->setText(fullPath);
+    
+    ui->m_showInExplorer->setEnabled(true);
+    ui->m_showInExplorer->disconnect();
+    connect(ui->m_showInExplorer, &QPushButton::clicked, this, [=]()
```

Comment:
do you need to capture anything you are calling a static method?

just so you are aware this will eventually be marked deprecated in C++20.    

https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2018/p0806r2.html
https://www.nextptr.com/tutorial/ta1430524603/capture-this-in-lambda-expression-timeline-of-change